schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INTEGER,
```

Comment:
- [ OPTIONAL ] I highly recommend to mark the id column as primary key to make it unique and not null 🙏🏻👍🏻✨✨